.githooks/README.md

```diff
@@ -0,0 +1,69 @@
+# Shared Git Hooks
+
+This directory contains git hooks that can be shared across the team to ensure consistent development workflows.
+
+## Installation
+
+To install these hooks on your local repository:
+
+```bash
+./.githooks/install.sh
+```
```

Comment:
DEVELOPING.md mentions installing using the make targets. seems confusing to have two different installation methods.